modules/simple-payments/simple-payments.css

```diff
@@ -0,0 +1,17 @@
+.jetpack-simple-payments__purchase-message {
+	display: none;
+	color: #fff;
```

Comment:
Just fyi I've changed to `div` instead of `p`;

```css
body .jetpack-simple-payments__wrapper div.jetpack-simple-payments__purchase-message {
     color: #fff;
}
```